lab_1_keywords_tfidf/main.py

```diff
@@ -99,7 +145,20 @@ def calculate_tfidf(term_freq: dict[str, float], idf: dict[str, float]) -> Optio
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if term_freq == {} or not isinstance(term_freq, dict) or not isinstance(idf, dict):
```

Comment:
to check if term_freq is empty, it is enough to write `if not term_freq`

Reply:
done